src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -107,25 +107,23 @@
 					AccessKey="X"
 					AutomationProperties.AutomationId="InnerNavigationToolbarCutButton"
 					Command="{x:Bind Commands.CutItem, Mode=OneWay}"
+          Content="{x:Bind Commands.CutItem.Icon}"
```

Comment:
Spacing looks off, can you run xaml styler?

Reply:
This was the github merge, i'll fix it

Reply:
Spacing still looks off.